src/notaryUtils.js

```diff
@@ -26,11 +26,15 @@ function createCaseID () {
  */
 function setupCase (params) {
   return co(function * () {
-    const uniqueID = params.caseID || uuid()
+    // the caseID param will be in the format of notary/cases/uuid,
+    // only check if its uuid is less than 40 characters
+    const uniqueID = params.caseID ? params.caseID.substring(params.caseID.indexOf('/cases/') + 7) : uuid()
```

Comment:
The length is a database restriction, see @lumberj's comment on a previous PR: https://github.com/interledger/five-bells-sender/pull/47#discussion_r56205805. Though that may be for the entire `caseID` URI not the "uniqueID", I am not sure. @lumberj could you provide some context?